src/AbstractFirestoreRepository.ts

```diff
@@ -25,43 +24,37 @@ import { ValidationError } from 'class-validator';
 export abstract class AbstractFirestoreRepository<T extends IEntity>
   extends BaseRepository
   implements IQueryBuilder<T>, IQueryExecutor<T> {
-  protected readonly subColMetadata: SubCollectionMetadata[];
+  protected readonly subColMetadata: CollectionMetadata[];
   protected readonly colMetadata: CollectionMetadata;
   protected readonly collectionType: FirestoreCollectionType;
   protected readonly colName: string;
-  protected readonly docId: string;
-  protected readonly subColName: string;
   protected readonly config: MetadataStorageConfig;
 
+  // TODO: Make this private when transactions have been fixed
+  public readonly collectionPath: string;
```

Comment:
I've made this public, as having the collection path can be a helpful workaround whilst transactions are being fixed :)